airbyte-config/models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -103,6 +103,8 @@
 
   String getSubmitterNumThreads();
 
+  String getJobsImagePullSecret();
```

Comment:
I didn't understand what is this for?

Reply:
oh ok it's:
> The injection of Kube image pull secret enables pulling jobs from private docker repositories that have docker authentication.

Would it be helpful to document what it is in the code too though?

Reply:
I will do so!

Reply:
done.